packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -75,10 +75,9 @@ export async function errorHandling(errorDetails: Error | string, label?: string
             if (imperativeError.mDetails.additionalDetails) {
                 const tokenError: string = imperativeError.mDetails.additionalDetails;
                 if (tokenError.includes("Token is not valid or expired.")) {
-                    if (this.isTheia()) {
-                        Gui.errorMessage(errToken).then(async () => {
-                            await Profiles.getInstance().ssoLogin(null, label);
-                        });
+                    if (isTheia()) {
```

Comment:
Thanks for fixing the `isTheia` call 🙂 
I believe the `isTheia` function may be duplicate code, since we already have `globals.ISTHEIA` that can be checked to determine whether we're running in a Theia environment.
Would it make sense to remove the `isTheia` function from this file and replace it with `globals.ISTHEIA`?